packages/fast-form-generator-react/src/form/form-item.children.tsx

```diff
@@ -189,65 +220,56 @@ class FormItemChildren extends React.Component<IFormItemChildrenProps & IManaged
         return components.length > 0 ? components : item;
     }
 
-    private getCurrentChildArray(currentChildren: JSX.Element): any[] {
+    private getCurrentChildArray(currentChildren: JSX.Element): React.ReactNode[] {
         return Array.isArray(currentChildren)
             ? currentChildren
-            : typeof currentChildren !== "undefined"
+            : typeof currentChildren !== "undefined" && currentChildren !== null
             ? [currentChildren]
             : [];
     }
 
-    private getChildComponent(item: any): JSX.Element {
-        return (
-            <item.component
-                key={uniqueId()}
-                {...generateExampleData(item.schema, "")}
-            />
-        );
+    private getChildComponent(item: any): IComponentData {
```

Comment:
is `item` really any?

Reply:
what it is is either a string or an interface with an id which is a string and props which is any so I can type it as that.